### PR TITLE
fix: app version information PT-1652

### PR DESCRIPTION
**Pull request by terovirtanen**
### Closes :no_good_woman:
**[PT-1652](https://helsinkisolutionoffice.atlassian.net/browse/PT-1652)** 

## Additional notes :spiral_notepad:

[PT-1652]: https://helsinkisolutionoffice.atlassian.net/browse/PT-1652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr302.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr302.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr302.dev.hel.ninja** :rocket::rocket::rocket:

File: src/domain/app/api/version.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    // const buildTime = process.env.BUILD_TIME ?? '';
+
+    // res.status(200).json({
+    //   status: 'ok',
+    //   release,
+    //   packageVersion,
+    //   commitHash,
+    //   buildTime,
+    // });
+
```
Comment (karisal-anders) [inline src/domain/app/api/version.ts:15]: IMHO Generally if code is commented out it should be removed—of course there are exceptions but I don't see any clear reason for keeping this code. If there is a very good reason for keeping this commented out code it should IMHO be given in comments as well.

File: src/domain/app/api/version.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    // REACT_APP_VERSION is set by react
+    const packageVersion = process.env.REACT_APP_VERSION ?? '';
+    const release = process.env.REACT_APP_RELEASE ?? '';
+    const commitHash = process.env.REACT_APP_COMMITHASH ?? '';
+    const buildTime = process.env.REACT_APP_BUILDTIME ?? '';
+
+    console.log(process.env);
```
Comment (karisal-anders) [inline src/domain/app/api/version.ts:22]: I assume this is a leftover debug code? It should be removed...

File: src/domain/app/api/version.ts (hunk 1 above)
Comment (terovirtanen) [reply src/domain/app/api/version.ts]: removed

File: src/domain/app/api/version.ts (hunk 2 above)
Comment (terovirtanen) [reply src/domain/app/api/version.ts]: removed

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr302.dev.hel.ninja** :rocket::rocket::rocket:

**Review by karisal-anders — APPROVED** [review]

**Comment by terovirtanen** [comment]
**kultus-admin-ui branch is deployed to platta: https://kultus-admin-ui-pr302.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**